### PR TITLE
Updated the comments in config.js file.

### PR DESCRIPTION
**Pull request by Jagatjot**
Solves Issue #120

### CODE REVIEW

**Comment by Jagatjot** [comment]
@dmoranj @fgalan here by just commenting the config file,the travis test is failing. Can you review this PR that closes Issue #120 and close the issue.

File: config.js [hunk 1]
```diff
@@ -22,53 +22,184 @@
...
+     */
     thinkingThingsPlugin: true,
+    /**
+     * QoS Level: at most once (0), at least once (1), exactly once (2). (default is 2).
+     */
     qos: 0,
+    /**
+     * Retain flag. (default is true). Normally if a publisher publishes a message to a topic, and no one is 
+     * subscribed to that topic(i.e retain flag is set to false) the message is simply discarded by the broker.
+     * The publisher can tell the broker to keep the last message on that topic by setting the retained message flag to true  .
```
Comment (fgalan) [inline config.js]: true  . -> true. (remove extra whitespace)

File: config.js [hunk 2]
```diff
@@ -22,53 +22,184 @@
...
+     */
     retain: false
 };
 
+/**
+ * Configuration for the AMQP binding.
+ */
 config.amqp = {
+    /**
+		 * Host where the AMQP broker is located.
```
Comment (fgalan) [inline config.js]: Indent in this line (and the next one) is not aligned with the previous.

File: config.js [hunk 3]
```diff
@@ -22,53 +22,184 @@
...
     queue: 'iotaqueue',
+    /**
+     * durable queue flag (default is false).
+     */
     options: {durable: true}
 };
 
 config.iota = {
+    /**
+	 * Configures the log level. Appropriate values are: FATAL, ERROR, INFO, WARN and DEBUG.
```
Comment (fgalan) [inline config.js]: Indent in this line (and the next one) is not aligned with the previous.

File: config.js [hunk 4]
```diff
@@ -22,53 +22,184 @@
...
+    /**
+     * Context Broker configuration. Defines the connection information to the instance of the Context Broker where
+     * the IoT Agent will send the device data.
+     */
     contextBroker: {
+        /**
+     * Host where the Context Broker is located.
+     */
         host: 'localhost',
+        /**
```
Comment (fgalan) [inline config.js]: Indent in this line is not aligned with the next one.

File: config.js [hunk 5]
```diff
@@ -22,53 +22,184 @@
...
+     */
     deviceRegistry: {
         type: 'mongodb'
     },
+    /**
+     * Mongo DB configuration section. This section will only be used if the deviceRegistry property has the type
+     * 'mongodb'.
+     */
     mongodb: {
+        /**
```
Comment (fgalan) [inline config.js]: Indent?

File: config.js [hunk 6]
```diff
@@ -22,53 +22,184 @@
...
+     * Port where the Context Broker is listening.
+     */
         port: '1026'
     },
+    /**
+     * Configuration of the North Port of the IoT Agent.
+     */
     server: {
+        /**
```
Comment (fgalan) [inline config.js]: Indent?

File: config.js [hunk 7]
```diff
@@ -22,53 +22,184 @@
...
+     * When this flag is active, the IoTAgent will add the TimeInstant attribute to every entity created, as well
+     * as a TimeInstant metadata to each attribute, with the current timestamp.
+     */
     timestamp: true,
+    /**
+     * Context Broker configuration. Defines the connection information to the instance of the Context Broker where
+     * the IoT Agent will send the device data.
+     */
     contextBroker: {
+        /**
```
Comment (fgalan) [inline config.js]: Indent?

**Comment by fgalan** [comment]
Thank you very much for the contribution! The idea is pretty good, please find only some minor comments regarding the added texts.

Do you plan do the same for the other IOTAs (UL, LWM2M, Sigfox, etc.)?

(Btw, I have re run the failing test in travis and now it works )

**Comment by Jagatjot** [comment]
@fgalan Thanks for your comments,  I have resolved the comments given by you. Yes, I can work on other IOTAs also. You can re-verify the changes and close this PR.

**Comment by Jagatjot** [comment]
@fgalan Added config.js file.

**Comment by fgalan** [comment]
Passing the ball to @dcalvoalonso for additional LGTM

**Comment by dcalvoalonso** [comment]
LGTM

**Pull request merged**